contributing.md

```diff
@@ -102,6 +102,23 @@ In some cases, `fqm-execution` will succeed in its calculation, but the data ret
 
 If everything from steps 1-3 seems correct, there may be an issue with the population processing logic in `fqm-execution`. In this case, please feel free to submit an issue or a pull request.
 
+### Population Results with Equivalent DateTime Comparisons
+
+A commonly seen unexpected population result may come up for when two compared `DateTime` properties are thought to be equivalent. When considering `DateTime` equivalencies, it is important to note the precision of the compared `DateTime` properties.
```

Comment:
```suggestion
A commonly seen unexpected population result may come up when two compared `DateTime` properties are thought to be equivalent. When considering `DateTime` equivalencies, it is important to note the precision of the compared `DateTime` properties.
```
I actually don't know but I think this sounds better? Grammar is not my specialty 😅

Reply:
I think the former is technically correct English, but this is definitely better English :-P